apps/academy-wep/components/knowledge-resources/ResourceCard.tsx

```
@@ -0,0 +1,119 @@
+"use client";
+
+import { motion } from "framer-motion";
+import Image from "next/image";
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+import { Clock, ThumbsUp, MessageSquare, Calendar } from "lucide-react";
+import { ResourceType } from "@/lib/knowledge-resources";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)